docs/v1.md

```diff
@@ -398,6 +452,7 @@
 - flows(): `OAuthFlows` | `undefined`
 - hasOpenIdConnectUrl(): `boolean`
 - openIdConnectUrl(): `string` | `undefined`
+- scopes(): `string[]` | `undefined`
```

Comment:
This is already in `SecurityRequirement`

Reply:
I'm removing this because it is already in the security requirement object, so afaik it is not really need here. I know this got simplified in v3 but still compatible. We don't need to mimic exactly what new versions do.

WDYT @derberg @fmvilas @jonaslagoni @magicmatatjahu